module-2/backend/week_one.md

```diff
@@ -5,30 +5,99 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
+    a. Get - used to view a page 
+    b. Post - Used to update a page 
+    c. Delete - deletes
+    d. Put - used to update a page
+    e. Patch - for partial updates
+2. What is Sinatra? 
+    Sinatra is a ruby framework to host programs on a server. In other words, its a tool used to allow a visual tool inside a 
+    browser. 
 4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
+    a. Model
+    b. View 
+    c. Controller 
+    This is the file structure used in sinatra/ruby programs
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes? 
+      Because it allows for easy understanding across all programs. It is saved time that would otherwise be spend figuring  
+      out what extension goes to what page. 
+6. What types of variables are accessible in our view templates without explicitly passing them? 
+      yield
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
-  get '/horses' do
+  get '/horses' do 
     erb :index
   end
   ```
-
-8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
-9. What's the purpose of ERB?
-10. Why do I need a development AND test database?
+    Add => @count = 1 above erb:index 
+    
+8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view? 
+    Add => name = "Mr. Ed"

```

Comment:
Part way there! But we also need to change the `erb :index` line to the following:

```ruby
get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```

Alternatively we could also do something like the following, which skips the step of declaring the local variable in the `get` method:

```ruby
get '/horses' do 
  erb :index, locals: {name: "Mr. Ed"}
end
```